### ISSUE TITLE
program doesn't work in windows 7

### ISSUE THREAD

**Issue by ranny70** [scubajorgen/TomTomWatch#40]
HI. I tried to install the program 
https://blog.studioblueplanet.net/software/tomtomwatch     (binary1.9)
but I can't run it on my computer with Win 7.
After opening the zip file none of the files inside run on win7.
 Is the program only for Linux?

If I'm in the wrong forum, can you tell me who to ask?

Thanks and sorry

**Comment by scubajorgen** [comment]
Mwah.... It is primarily designed for Windows so it should run :-) You need a Java Runtime Environment (JRE), so don't expect an executable inside the zip

**Comment by scubajorgen** [comment]
Yup, when pressing the left download. If nothing appears, read FAQ

⁣Get BlueMail for Android ​

**Comment by ranny70** [comment]
sorry.. but where are the FAQ?

Everything works for me and I'm very happy. I just can't read the average speed :-(

**Comment by ranny70** [comment]
FAQ!!  I found them :-)

**Comment by ranny70** [comment]
Is it possible to read the average speed?
